SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialTraceEventBuilder.cpp

```diff
@@ -116,9 +116,16 @@ FSpatialTraceEvent FSpatialTraceEventBuilder::CreateSendRPC(const UObject* Objec
 		.GetEvent();
 }
 
-FSpatialTraceEvent FSpatialTraceEventBuilder::CreateRPCLinearTraceEvent(const EventTraceUniqueId& LinearTraceId)
+FSpatialTraceEvent FSpatialTraceEventBuilder::CreateSendRPCLinearTraceEvent(const EventTraceUniqueId& LinearTraceId)
 {
-	return FSpatialTraceEventBuilder(GDK_EVENT_NAMESPACE "rpc_linear_trace")
+	return FSpatialTraceEventBuilder(GDK_EVENT_NAMESPACE "send_rpc_linear_trace")
```

Comment:
This was a recommendation on the previous PR after it was merged

Reply:
Nicu!